core/src/main/java/com/alibaba/nacos/core/utils/WebUtils.java

```diff
@@ -43,6 +43,18 @@
  */
 public class WebUtils {
     
+    private static final String ENCODING_KEY = "encoding";
+    
+    private static final String ACCEPT_CHARSET_KEY = "Accept-Charset";
```

Comment:
Can be replaced with
`com.alibaba.nacos.common.constant.HttpHeaderConsts#ACCEPT_CHARSET`

Reply:
done